### PR TITLE
[LUS-87] Two Column Content Card

### PR DESCRIPTION
**Pull request by YolineB**
# Pull Request Process

## Copy and Paste / Write ticket description
[Fixes LUS-87 ](https://linear.app/ofashandfire/issue/LUS-87/content-component-two-column-content-card)Two Column Component

[Figma](https://www.figma.com/file/NOsfGFCFF0yOR8FeiHwhZy/Website-Redesign?node-id=228%3A17125&mode=dev)

## Describe the changes you've made to resolve the issue

- Added TwoColumn.tsx 

## Code Review Checklist

- [ ] Have the package.json or package-lock.json been changed? If so, does the pull request description say why?
- [ ] Has the UI for mobile and desktop been changed, did you test on IphoneXr and desktop views?
- [ ] Does the UI match the requirements in the ticket and designs in figma; does it pass the visual smell test?
- [ ] Have TypeScript files been changed, does the TypeScript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: app/components/ui/Typography/Title.tsx [hunk 1]
```diff
@@ -23,14 +23,16 @@ export const Title3 = ({ children, className = "" }: TitleProps) => {
 };
 
 export function TitleComponent({ title, titleSize, bold }: TitleProps) {
+  const boldClass = bold ? "md:font-bold" : "md:font-normal";
```
Comment (jtorreggiani) [inline app/components/ui/Typography/Title.tsx:26]: Small refactor to move this logic into the TitleComponent itself. So we don't need to compute this in every place we use it.

File: app/utils/contentful.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export async function getEntryById(id: string) {
 export async function getPage() {
   const entry = await client.getEntry(LANDING_PAGE_ID, {
     include: 2,
-    locale: "es-US",
+    locale: "en-US",
```
Comment (jtorreggiani) [inline app/utils/contentful.ts:24]: I had hardcode this value to test that translations work. Reverting for now. We need to make this dynamic in the future.

File: app/components/PageSection/MultiCardLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+// Next components
+import Image from "next/image";
+
+// Types
+import { PageSectionType } from "@/app/constants/types";
+
+// TODO: Refactor to be used in Research Centers section.
+export default function MultiCardLayout({
```
Comment (jtorreggiani) [inline app/components/PageSection/MultiCardLayout.tsx:8]: We need to keep working on this one.

**Review by jtorreggiani — APPROVED** [review]
There were some conflicts in the design and purpose for the "two column layout" and the "two column content cards". I pushed up some organization changes to make these more clear. We still need to finish the design for the content cards, but I think we should do that in a separate PR since this one great to accommodate the over arching changes.

File: app/utils/contentful.ts (hunk 2 above)
Comment (YolineB) [reply app/utils/contentful.ts:24]: Created ticket [LUS-105 Dynamic Locale Switching](https://linear.app/ofashandfire/issue/LUS-105/dynamic-locale-switching-en-es)

**Pull request merged**